ansible/reset-net.yml

```diff
@@ -64,6 +75,7 @@
 
     - name: Start epoch deamon
       command: "{{ epoch_bin }} start"
+      when: epoch.stat.exists
```

Comment:
If epoch bin does not exist I would rather expect script to have failed and not to arrive at this point. Would it make sense to include explicit fail steps early in the playbook checking for the preconditions? (I would expect script to fail explicitly - rather than skipping tasks - if epoch bin is absent or epoch config is absent.)

Reply:
This is just a linter fix/hack.

see https://github.com/aeternity/infrastructure/pull/17/files#diff-c7b2f1005a3b723ea1cfc920d97fea1cR15